### DISCUSSION TITLE
Trouble accepting double quotes between single quotes

### DISCUSSION THREAD

**Discussion by felicksLindgren** [JoshClose/CsvHelper#1722] [Q&A] [answered]
Hello,

I'm trying to read some data from a csv.

This is the structure of my data:
'EL1076042'$'6010000002001'$'Standardsortiment. (B)'$'GUMMIKABEL SORT 3X1"'

This is my configuration as of now:
`var config = new CsvConfiguration(CultureInfo.InvariantCulture)
            {
                Delimiter = "$",
                Quote = '\'',
                Mode = CsvMode.Escape
            };`

My issue is that I want to accept double quotes inside a string e.g. 'This is 3"' but can't seem to make it work when looking around the documentation and with this configuration.

Any ideas on how to accept double qoutes inside my strings?

**Comment by felicksLindgren** [comment, marked as answer]
UPDATE!

`var config = new CsvConfiguration(CultureInfo.InvariantCulture)
            {
                Delimiter = "$",
                Quote = '\'',
                Escape = '\'',
                Mode = CsvMode.RFC4180
            };`

With this configuration, it seems to be handling the double qoutes inside my strings just fine.
So consider this question to be answered as of now.

**Reply by JoshClose** [reply]
FYI, mode `RFC4180` is the default.

**Reply by felicksLindgren** [reply]
Was the new escape character that fixed it
